Formula/bcoin.rb

```diff
@@ -0,0 +1,33 @@
+require "language/node"
+
+class Bcoin < Formula
+  desc "Javascript bitcoin library for node.js and browsers"
+  homepage "https://bcoin.io"
+  url "https://github.com/bcoin-org/bcoin/archive/v2.0.0.tar.gz"
+  sha256 "ff99b735d3b23624455d72eada0de7f7ba4207f4a4cc0cab963ca2036de254a3"
+  depends_on "python" => :build
+  depends_on "node@10"
```

Comment:
Please try and get it to build with the latest node.js. Using old versions in new formulae should be avoided.

Reply:
Updated to use `node`